static/js/stream_data.js

```diff
@@ -523,15 +531,23 @@ export function id_is_subscribed(stream_id) {
 export function get_stream_privacy_policy(stream_id) {
     const sub = sub_store.get(stream_id);
 
-    if (!sub.invite_only) {
+    if (!sub.invite_only && !sub.is_web_public) {
         return stream_privacy_policy_values.public.code;
     }
     if (sub.invite_only && !sub.history_public_to_subscribers) {
         return stream_privacy_policy_values.private.code;
     }
+    if (sub.is_web_public) {
+        return stream_privacy_policy_values.web_public.code;
+    }
     return stream_privacy_policy_values.private_with_public_history.code;
 }
 
+export function is_web_public(stream_id) {
+    const sub = sub_store.get(stream_id);
+    return sub !== undefined && sub.is_web_public;
+}
```

Comment:
Can we readily tweak the dropdown code to not display the option when not in a development environment?  If so, we can merge this to head off merge conflicts.